src/target/jtag_devs.c

```diff
@@ -25,35 +26,45 @@
 
 jtag_dev_descr_t dev_descr[] = {
 	{.idcode = 0x0BA00477, .idmask = 0x0FFF0FFF,
-		.descr = "ARM Limited: ADIv5 JTAG-DP port.",
+		.descr = "ADIv5 JTAG-DP port.",
 		.handler = adiv5_jtag_dp_handler},
+	{.idcode = 0x00000477, .idmask = 0x00000FFF,
+		.descr = "Unknown ARM."},
 	{.idcode = 0x06410041, .idmask = 0x0FFFFFFF,
-		.descr = "ST Microelectronics: STM32, Medium density."},
+		.descr = "STM32, Medium density."},
 	{.idcode = 0x06412041, .idmask = 0x0FFFFFFF,
-		.descr = "ST Microelectronics: STM32, Low density."},
+		.descr = "STM32, Low density."},
 	{.idcode = 0x06414041, .idmask = 0x0FFFFFFF,
-		.descr = "ST Microelectronics: STM32, High density."},
+		.descr = "STM32, High density."},
 	{.idcode = 0x06416041, .idmask = 0x0FFFFFFF,
-		.descr = "ST Microelectronics: STM32L."},
+		.descr = "STM32L."},
 	{.idcode = 0x06418041, .idmask = 0x0FFFFFFF,
-		.descr = "ST Microelectronics: STM32, Connectivity Line."},
+		.descr = "STM32, Connectivity Line."},
 	{.idcode = 0x06420041, .idmask = 0x0FFFFFFF,
-		.descr = "ST Microelectronics: STM32, Value Line."},
+		.descr = "STM32, Value Line."},
 	{.idcode = 0x06428041, .idmask = 0x0FFFFFFF,
-		.descr = "ST Microelectronics: STM32, Value Line, High density."},
+		.descr = "STM32, Value Line, High density."},
 	{.idcode = 0x06411041, .idmask = 0xFFFFFFFF,
-		.descr = "ST Microelectronics: STM32F2xx."},
+		.descr = "STM32F2xx."},
 	{.idcode = 0x06422041, .idmask = 0xFFFFFFFF,
-		.descr = "ST Microelectronics: STM32F3xx."},
-	{.idcode = 0x06413041 , .idmask = 0xFFFFFFFF,
-		.descr = "ST Microelectronics: STM32F4xx."},
-	{.idcode = 0x0BB11477 , .idmask = 0xFFFFFFFF,
+		.descr = "STM32F3xx."},
+	{.idcode = 0x06413041, .idmask = 0xFFFFFFFF,
+		.descr = "STM32F4xx."},
+	{.idcode = 0x00000041, .idmask = 0x00000FFF,
+		.descr = "STM32 BSD."},
+	{.idcode = 0x0BB11477, .idmask = 0xFFFFFFFF,
 		.descr = "NPX: LPC11C24."},
-	{.idcode = 0x4BA00477 , .idmask = 0xFFFFFFFF,
+	{.idcode = 0x4BA00477, .idmask = 0xFFFFFFFF,
 		.descr = "NXP: LPC17xx family."},
+	{.idcode = 0x00000093, .idmask = 0x00000FFF,
+		.descr = "Xilinx."},
+	{.idcode = 0x0000063D, .idmask = 0x00000FFF,
+		.descr = "Xambala: RVDBG013."},
+	{.idcode = 0x000007A3, .idmask = 0x00000FFF,
+		.descr = "Gigadevice BSD."},
 /* Just for fun, unsupported */
-	{.idcode = 0x8940303F, .idmask = 0xFFFFFFFF, .descr = "ATMEL: ATMega16."},
-	{.idcode = 0x0792603F, .idmask = 0xFFFFFFFF, .descr = "ATMEL: AT91SAM9261."},
+	{.idcode = 0x8940303F, .idmask = 0xFFFFFFFF, .descr = "Atmel: ATMega16."},
```

Comment:
If we are removing the manufacturer description above for the ST parts then we probably should remove Atmel here? (and on the line below)

Reply:
Sure, we can do that